Vic2ToHoI4/Data_Files/blankMod/output/interface/culture_ideas.gfx

```diff
@@ -1,2423 +1,1870 @@
 spriteTypes = {
 	spriteType = {
 		name = "GFX_idea_culture_aborigine"
-		texturefile = "gfx/interface/culture/culture_aborigine.dds"
+		texturefile = gfx/interface/culture/culture_aborigine.dds"
+	}
+	spriteType = {
+		name = "GFX_idea_culture_adelphic"
+		texturefile = gfx/interface/culture/culture_adelphic.dds"
 	}
-
 	spriteType = {
 		name = "GFX_idea_culture_afar"
-		texturefile = "gfx/interface/culture/culture_afar.dds"
+		texturefile = gfx/interface/culture/culture_afar.dds"
+	}
+	spriteType = {
+		name = "GFX_idea_culture_afer"
+		texturefile = gfx/interface/culture/culture_afer.dds"
 	}
-
 	spriteType = {
-		name = "GFX_idea_culture_africaine"
-		texturefile = "gfx/interface/culture/culture_africaine.dds"
+		name = "GFX_idea_culture_africaine "
```

Comment:
Spaces like this are probably not good. Make sure they're all removed.

Reply:
Solved